### PR TITLE
macOS client is not able to do auto updates

### PR DESCRIPTION
**Pull request by FlexW**
Signed-off-by: Felix Weilbach <felix.weilbach@nextcloud.com>

### CODE REVIEW

File: doc/autoupdate.rst [hunk 1]
```diff
@@ -32,10 +32,9 @@ itself. Should the silent update fail, the client offers a manual download.
 macOS
 ^^^^^
 
-If a new update is available, the Nextcloud client initializes a pop-up dialog
-to alert you of the update and requesting that you update to the latest
-version. Due to their use of the Sparkle frameworks, this is the default
-process for macOS applications.
+There is no automatic updater on macOS. If a new update is available,
+the Nextcloud client initializes a pop-up dialog to alert you of the
+update and requesting that you update to the latest version.
```
Comment (allexzander) [inline doc/autoupdate.rst:37]: @FlexW 
`the latest version.` -> `the latest version` **manually**`.`

I guess this would be more precise?

Comment (mgallien) [reply doc/autoupdate.rst:37]: I agree

**Review by mgallien — CHANGES_REQUESTED** [review]
see my reply to Alex's comment

**Review by mgallien — APPROVED** [review]
thanks

**Review by allexzander — APPROVED** [review]

**Comment by JMoVS** [comment]
I was still not able to click on the download link on 3.3.5 - is this intentional?

**Pull request merged**